tests/test_utils.ts

```diff
@@ -426,7 +426,8 @@ async function spawnServer(
   // @ts-ignore yes it does
   for await (const line of lines.values({ preventCancel: true })) {
     output.push(line);
-    const match = line.match(/https?:\/\/localhost:\d+/g);
+    const match = line.match("Fresh ready");
```

Comment:
Since the hostname/IP can now be variable this needed to be changed to assert the fresh is running. "Fresh ready" seemed to be better as it'll always be there and not dynamically change.